GETTING_STARTED.md

```diff
@@ -119,7 +119,7 @@ It is also possible to explicitly specify the class:
 
 ```ruby
 # This will use the User class (otherwise Admin would have been guessed)
-factory :admin, class: User
+factory :admin, class: 'User'
```

Comment:
Let's switch to double quotes to match the [thoughtbot style guide](https://github.com/thoughtbot/guides/tree/master/style/ruby)
```suggestion
factory :admin, class: "User"
```